tox.ini

```diff
@@ -84,8 +89,24 @@ commands =
 
 [testenv:flake8]
 basepython = python2.7
-deps = flake8
-commands = flake8 --doctests src tests setup.py
+deps =
+    flake8
+    flake8-html
```

Comment:
yes we do not allow flake8 violations, but testing and tools should help to find and remove violations and html reports could help here.